dataactbroker/handlers/fileHandler.py

```diff
@@ -297,8 +301,16 @@ def getStatus(self):
             for jobId in jobs:
                 jobInfo = {}
                 jobType = self.jobManager.getJobType(jobId)
+
                 if jobType != "csv_record_validation" and jobType != "validation":
                     continue
+
+                # TODO Skip D1 file until validation is added, remove these lines once D1 validation is added
```

Comment:
I will add my notes on the D1 changes that are needed to DB-925, I have a list of the places that got these TODO notes.